src/go/rpk/pkg/cli/cmd/wasm/generate.go

```diff
@@ -39,36 +39,20 @@ var manifest = func() map[string][]genFile {
 }
 
 func NewGenerateCommand(fs afero.Fs) *cobra.Command {
-	var dir string
 	const localDir = "./"
 	command := &cobra.Command{
 		Use:	"generate",
 		Short:	"Create a npm template project for inline WASM engine",
 		RunE: func(_ *cobra.Command, args []string) error {
-			if dir == localDir {
-				path, err := filepath.Abs(dir)
-				if err != nil {
-					return err
-				}
-				rootPath := filepath.Join(path, "wasm")
-				return executeGenerate(fs, rootPath)
-
-			}
-			path, err := filepath.Abs(dir)
+			path, err := filepath.Abs(localDir)
+			rootPath := filepath.Join(path, "wasm")
```

Comment:
I thought this was supposed to fix the unexpected behavior of creating a project named "wasm" by default

Reply:
this is the first commit on this series, I needed to keep that line because I want to pass all tests on that commit, but the second commit removes it.